planner/core/logical_plans.go

```diff
@@ -371,10 +371,12 @@ type DataSource struct {
 // accessPath indicates the way we access a table: by using single index, or by using multiple indexes,
 // or just by using table scan.
 type accessPath struct {
-	index      *model.IndexInfo
-	idxCols    []*expression.Column
-	idxColLens []int
-	ranges     []*ranger.Range
+	index          *model.IndexInfo
+	fullIdxCols    []*expression.Column
+	fullIdxColLens []int
+	idxCols        []*expression.Column
+	idxColLens     []int
```

Comment:
Can we wrap up `[]*expression.Column` and `[]int` as a struct?

Reply:
Yeah, i want to do this, but not sure in this pr or in future pr.